File: PlaywrightCSharp/TestExample.cs
using System.Text.RegularExpressions;
using System.Threading.Tasks;
using Microsoft.Playwright;
using Microsoft.Playwright.MSTest;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace PlaywrightTests
{
    [TestClass]
    public class TestExample : PageTest
    {
        [TestMethod]
        public async Task ShouldClickButton()
        {
            await Page.GotoAsync("data:text/html,<script>var result;</script><button onclick='result=\"Clicked\"'>Go</button>");
            await Page.GetByRole(AriaRole.Button).ClickAsync();
            Assert.AreEqual("Clicked", Page.EvaluateAsync<string>("result").Result);
        }

        [TestMethod]
        public async Task ShouldCheckTheBox()
        {
            await Page.SetContentAsync("<input id='checkbox' type='checkbox'></input>");
            await Page.GetByRole(AriaRole.Checkbox).ClickAsync();
            Assert.IsTrue(Page.EvaluateAsync<bool>("() => window['checkbox'].checked").Result);
        }

        [TestMethod]
        public async Task ShouldSearchWiki()
        {
            await Page.GotoAsync("https://www.wikipedia.org/");
            await Page.SelectOptionAsync("#searchLanguage", "English");
            await Page.Locator("input[name=\"search\"]").ClickAsync();
            await Page.Locator("input[name=\"search\"]").FillAsync("Playwright (software)");
            await Page.Locator("input[name=\"search\"]").PressAsync("Enter");
            Assert.AreEqual("https://en.wikipedia.org/wiki/Playwright_(software)", Page.Url);
        }
    }
}

File: PlaywrightPython/test_example.py
import re
from playwright.sync_api import Page, expect

def test_should_click_button(page: Page):
    page.goto('data:text/html,<script>var result;</script><button onclick="result=\'Clicked\'">Go</button>')
    page.locator('button').click()
    result = page.evaluate('result')
    assert result == 'Clicked'

def test_should_check_the_box(page: Page):
    page.set_content('<input id="checkbox" type="checkbox"></input>')
    page.locator('input').check()
    is_checked = page.evaluate('() => window.checkbox.checked')
    assert is_checked is True

def test_should_search_wiki(page: Page):
    page.goto('https://www.wikipedia.org/')
    page.select_option('#searchLanguage', 'en')
    page.locator('input[name="search"]').click()
    page.locator('input[name="search"]').fill('Playwright (software)')
    page.locator('input[name="search"]').press('Enter')
    page.wait_for_url('https://en.wikipedia.org/wiki/Playwright_(software)')
    expect(page).to_have_url('https://en.wikipedia.org/wiki/Playwright_(software)')
